core-web/libs/block-editor/src/lib/extensions/image-tabview-form/image-tabview-form.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="tabview-container">
+    <ng-container [formGroup]="form">
+        <p-tabView>
+            <p-tabPanel header="dotCMS" leftIcon="pi pi-images">
+                <div class="search-box">
+                    <span class="p-input-icon-right">
+                        <input
+                            class="search"
+                            #inputSearch
+                            type="text"
+                            pInputText
+                            formControlName="search"
+                            placeholder="Search"
```

Comment:
how we handle internationalization here ?

Reply:
We have to create an issue for that. Nothing in the editor is being internationalized.